Content Added

justThoughtless opened the pull request:
sac data filling

justThoughtless committed:
about us

justThoughtless committed:
about us

justThoughtless committed:
navbar undo

justThoughtless committed:
navbar undo

justThoughtless committed:
structure added

justThoughtless committed:
structure added

subhalingamd reviewed (changes requested):
Aren't there many positions?

justThoughtless committed:
team added

justThoughtless committed:
team added

justThoughtless committed:
team imagees added

justThoughtless committed:
team imagees added

justThoughtless committed:
mailto and target blank adddedto links

justThoughtless committed:
mailto and target blank adddedto links

subhalingamd reviewed (commented):
Nice default icon if image is missing 👍

justThoughtless committed:
facebook and email added

justThoughtless committed:
facebook and email added

justThoughtless committed:
Minimum distance in minutes changed and sac minutes added

justThoughtless committed:
Minimum distance in minutes changed and sac minutes added

justThoughtless committed:
Arrows for minutes scroll disbled in sac bcoz of only 2 minute years

justThoughtless committed:
Arrows for minutes scroll disbled in sac bcoz of only 2 minute years

justThoughtless committed:
Initiaves added to sac

justThoughtless committed:
Initiaves added to sac

subhalingamd reviewed (commented):
For SAC Minutes

subhalingamd reviewed (changes requested):
For Initiatives

justThoughtless committed:
bhm data added no minutes on drive so commented the section

justThoughtless committed:
bhm data added no minutes on drive so commented the section

justThoughtless committed:
brca about added no minutes or initiatives on drive so commented

justThoughtless committed:
brca about added no minutes or initiatives on drive so commented

justThoughtless committed:
BHM structure added and BSA content added BSA Minutes not availabale therfore commented

justThoughtless committed:
BHM structure added and BSA content added BSA Minutes not availabale therfore commented

justThoughtless committed:
merged

justThoughtless committed:
boards data entered except bsa structure and bhm structure need approval

justThoughtless committed:
bsa strcture added

subhalingamd reviewed (changes requested):
**BHM**
- [ ] Link Constituion & change text
- [ ] Contact us not linked *(Note: Email ID has to be changed in `href`)*
- [ ] Change website link to `https://home.iitd.ac.in/bhm.php`

subhalingamd reviewed (changes requested):
**BSA**
- [ ] Link Constituion & change text
- [ ] Contact us not linked *(Note: Email ID has to be changed in `href`)*
- [ ] Change website link to BSA site

justThoughtless committed:
all constitution pdf linked

justThoughtless committed:
a link within li tab in sacpage

justThoughtless committed:
display changed to block in initialtives list

justThoughtless committed:
extra comment ending removed

justThoughtless committed:
clubs logos and facebook links added

subhalingamd committed:
Update style.css

justThoughtless committed:
Sac page info and mid term report updated

justThoughtless committed:
about sac updated

justThoughtless committed:
bhm,bsa,sac end term reports added

justThoughtless committed:
sac structure changed

justThoughtless committed:
strcture changed

justThoughtless committed:
style changes

justThoughtless committed:
clubs website added

subhalingamd committed:
Changes in SAC

- [x] remove unused year
- [x] consti heading
- [x] fb link
- [x] flip report book
- [x] mid TERM ~year~
- [x] sac about bullet inline
- [x] 20jul > 20 sept
- [x] year add ' ('20)

justThoughtless committed:
Merge branch 'sac_data' of https://github.com/ashugupta249/sac_website into sac_data

justThoughtless committed:
Consti updated

justThoughtless committed:
Consti text updated

justThoughtless committed:
project seeking link added

justThoughtless committed:
clubs links added

subhalingamd committed:
sac initatives pg no changes

subhalingamd committed:
Re: update sac/initiatives

click anywhere to open the link

subhalingamd committed:
sac policy changes content

justThoughtless committed:
Merge branch 'sac_data' of https://github.com/ashugupta249/sac_website into sac_data

subhalingamd committed:
Consti spell fix in sac

justThoughtless committed:
structure images

justThoughtless committed:
Merge branch 'sac_data' of https://github.com/ashugupta249/sac_website into sac_data

justThoughtless committed:
structure added

justThoughtless committed:
clubs added and logos fixed

subhalingamd committed:
update structure (+css)

subhalingamd committed:
update contact details for boards

subhalingamd committed:
remove https in footer links for boards

justThoughtless committed:
list number removed

justThoughtless committed:
Merge branch 'sac_data' of https://github.com/ashugupta249/sac_website into sac_data

justThoughtless committed:
padding removed

justThoughtless committed:
arrow in sac fixed:

subhalingamd committed:
Fix external link not opening in club cards

subhalingamd committed:
favicon added

subhalingamd committed:
'year' to 'term' in boards initiatives

justThoughtless committed:
AAIP updated

justThoughtless committed:
merge fix

subhalingamd committed:
Update footer

subhalingamd committed:
remove ena

subhalingamd committed:
change 'structure & responsibilities' to 'Structure of the Student Team'

subhalingamd committed:
prevent hash change when click on modal close in clubs

subhalingamd committed:
comment instagram link

subhalingamd committed:
modify facilities card img in home

subhalingamd committed:
re: remove ena

subhalingamd committed:
Update text for clubs & projects in home

justThoughtless committed:
text align and top margin in explore home

subhalingamd committed:
modify font-weight for projects

justThoughtless committed:
Margin added to explore text

subhalingamd committed:
remove consti text for sac & bsp

subhalingamd committed:
consti & tabs modified

justThoughtless committed:
sac team updated

justThoughtless committed:
bsw end term report added:

justThoughtless committed:
endterm>midterm

subhalingamd committed:
Test font size for mobiles

subhalingamd committed:
Requested changes in footer

9. Footer links - CSC actual link - https://csc.iitd.ac.in/

10. Footer links - eCertification Link - ecertification.iitd.ac.in

subhalingamd committed:
speed up toggle-tab opening

subhalingamd committed:
activities content update

subhalingamd committed:
minor change in dev area

subhalingamd committed:
add theme color for address bar

subhalingamd committed:
facilities coming soon

justThoughtless committed:
bhm hostel reports, SAC structure, boards order in navbar

justThoughtless committed:
unknown arrows below reports removed

justThoughtless committed:
minutes section added to boards

subhalingamd committed:
Revert "speed up toggle-tab opening"

This reverts commit b4a017ce8329618908e8ff8e243eefce2d15dd5d.

justThoughtless committed:
boards reports added

subhalingamd committed:
Merge branch 'master' into sac_data
